### DISCUSSION TITLE
putting an image instead of node

### DISCUSSION THREAD

**Discussion by omeryilmazinfo** [bcakmakoglu/vue-flow#1098] [Q&A] [answered]
Is it possible to put an image (png, jpg) instead of the node?  Thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
Well, a custom node can be anything that is valid HTML so yeah, you can use an image as a node if you want to.

[Here's an example.](https://codesandbox.io/p/sandbox/ecstatic-violet-j5q2mv?file=%2Fsrc%2FCustomNode.vue%3A1%2C1-21%2C1)
